xAPI.md

```diff
@@ -2948,6 +2948,13 @@ do not match.
 
 * The LRS MAY respond before Statements that have been stored are available for retrieval.
 
+###### Activity Provider Requirements
+
+* Activity Providers SHOULD use POST instead of PUT. 
```

Comment:
I'd like it; basically, to me the preference order goes: POST with IDs > PUT > POST without IDs